XpressAI/x-template-speech_recognition
# ✨ Add silero model inference

Welcome to Xircuits! Thank you for making a pull request. Please ensure that your pull request follows the template.

# Description

This PR allows you to use pretrained Silero model to output the text based on speech from an audio file. 

## Pull Request Type

- [x] Xircuits Template

## Type of Change


# Tests

1. Clone this repository
2. Create virtual environments and install the required python packages.
```
pip install -r requirements.txt
```
3. Run xircuits from the root directory
```
xircuits
```
4. Download or record an audio file and place it in the root folder, and change the path to the audio file in the input of the SileroModelInference component. 

5. Run the xircuits file (SileroInference.xircuits) and ensure it is able to finish execution without any errors.


## Tested on?
